frontend/components/panels/VaultLiquidationLimitsCheckPanel.stories.js

```diff
@@ -0,0 +1,49 @@
+import { storiesOf } from '@storybook/vue';
+import faker from 'faker';
+import BigNumber from 'bignumber.js';
+import VaultLiquidationLimitsCheckPanel from './VaultLiquidationLimitsCheckPanel';
+import { generateFakeVaultNotLiquidatedTransaction } from '~/helpers/generateFakeVault';
+
+const fakeVaultLiquidationTransaction = generateFakeVaultNotLiquidatedTransaction();
```

Comment:
Why don't you just generate a fake collateral 

```ts 
import { getAllCollateralTypes } from 'auctions-core/src/constants/COLLATERALS.ts';

const COLLATERALS = getAllCollateralTypes();
faker.helpers.randomize(COLLATERALS)
```


and then export the function to generate mock `LiqudationLimits`? That seems a lot easier than all this.